src/tracker/psmove_tracker.c

```diff
@@ -401,7 +379,21 @@ psmove_tracker_remember_color(PSMoveTracker *tracker, struct PSMove_RGBValue rgb
 
 // -------- END: internal functions only
 
+void
+psmove_tracker_settings_set_default(PSMoveTrackerSettings *settings)
+{
+    *settings = tracker_default_settings;  // Doesn't work.
```

Comment:
I..err...actually, don't know. I'm fairly sure it does work, since I'm using it, so the comment is probably wrong. Will have a look tonight. Maybe @cboulay has some idea?